classes/migration/upgrade/v3_5_0/I7135_CreateNewRorRegistryCacheTables.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @file classes/migration/upgrade/v3_5_0/I7135_CreateNewRorRegistryCacheTables.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I7135_CreateNewRorRegistryCacheTables
+ *
+ * @brief Describe database table structures.
+ */
+
+namespace PKP\migration\upgrade\v3_5_0;
+
+use Illuminate\Database\Query\JoinClause;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema as Schema;
+use PKP\facades\Repo;
+use PKP\install\DowngradeNotSupportedException;
+use PKP\migration\Migration;
+use PKP\task\UpdateRorRegistryDataset;
+
+class I7135_CreateNewRorRegistryCacheTables extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('rors', function (Blueprint $table) {
+            $table->comment('Ror registry dataset cache');
+            $table->bigInteger('ror_id')->autoIncrement();
+            $table->string('ror')->nullable(false);
+            $table->string('display_locale', 28)->default('');
```

Comment:
Hi @GaziYucel , I encountered an SQL error regarding this column having only a length of 28. In my case, the value being set is "no_lang_code: HUCH, HUCS, HUS, HYKS" which contains 35 characters, so my migration was unsuccessful for a specific record included in the records. Can you please check? Thanks!